packages/nuxeo-amazon-s3-package/src/main/resources/install/templates/s3binaries/nuxeo.defaults

```diff
@@ -5,6 +5,7 @@ s3binaries.target=.
 # mandatory
 nuxeo.s3storage.bucket=
 nuxeo.core.binarymanager=org.nuxeo.ecm.core.storage.sql.S3BinaryManager
+nuxeo.core.blobstore.keyStrategy=managed
```

Comment:
Is this configuration change is compatible with an upgrade? ie: it will be transparent for the users?

Reply:
switching from `digest` to `managed` should work fine as we're basically just trusting the client and letting them use any key. We could leave it as `digest` by default but then uploads > 5GBs won't work... not until [NXP-29517|https://jira.nuxeo.com/browse/NXP-29517] is done. OTOH we can't easily migrate back from `managed` to `digest` 🤔  so perhaps we should indeed leave it as `digest` for now